components/uniset/src/lib.rs

```diff
@@ -53,7 +53,7 @@
 mod builder;
 mod conversions;
 pub mod enum_props;
-pub mod props;
+mod props;
```

Comment:
As long as we have the comment telling people not to use this module, I don't see a reason to change the visibility of `props`, since doing so incurs costs on props_test.rs as Zibi noted above.

I think the main thing is that we shouldn't ship provider_ppucd onto crates.io.

Reply:
That sounds good -- I've reverted the visibility from private back to public. Is there any additional code change required to prevent shipping provider_ppucd onto crates.io? Or is that achieved by the lack of some configuration or lack of manual action?

Reply:
I don't know.  @zbraniecki, is there a flag we can set to prevent provider_ppucd from being pushed to crates.io?

Reply:
nah, we don't have automated shipping yet. I may look into this next week but so far we manually ship crates. The only question is which crates are linked from `icu` meta crate.

Reply:
Follow-up: #636